flink/v1.12/flink/src/main/java/com/netease/arctic/flink/table/ArcticDynamicSource.java

```diff
@@ -235,6 +238,16 @@ public boolean supportsNestedProjection() {
 
   @Override
   public void applyProjection(int[][] projectedFields) {
```

Comment:
The projection logic here should also apply to com.netease.arctic.flink.table.ArcticFileSource